### PR TITLE
docs(cn): translate docusaurus/docs/developing-components-in-isolation.md into Chinese

### PR DESCRIPTION
**Pull request by liweina-fans**
Translate docusaurus/docs/developing-components-in-isolation.md into Chinese.

### CODE REVIEW

File: docusaurus/docs/developing-components-in-isolation.md [hunk 1]
```diff
@@ -1,59 +1,57 @@
...
+也可以将 Stroybook 或样式指南部署为静态应用程序。这样，团队中的每个人都可以查看和检查 UI 组件的不同状态，而无需启动后端服务器或在应用程序中创建帐户。
 
-## Getting Started with Storybook
+## Storybook 入门
 
-Storybook is a development environment for React UI components. It allows you to browse a component library, view the different states of each component, and interactively develop and test components.
-
-Run the following command inside your app’s directory:
+Storybook 是 React UI 组件的开发环境。它允许你浏览组件库，查看每个组件的不同状态，并交互式地开发和测试组件。
+在应用程序的目录中运行以下命令：
```
Comment (QC-L) [inline docusaurus/docs/developing-components-in-isolation.md]: 请不要删除多余的空行

File: docusaurus/docs/developing-components-in-isolation.md [hunk 2]
```diff
@@ -1,59 +1,57 @@
 ---
 id: developing-components-in-isolation
-title: Developing Components in Isolation
+title: 独立开发组件
 ---
 
-Usually, in an app, you have a lot of UI components, and each of them has many different states.
-For an example, a basic button component could have the following states:
+通常在一个应用程序中会有许多 UI 组件，每个组件都有不同的状态。
```
Comment (QC-L) [inline docusaurus/docs/developing-components-in-isolation.md]:
```suggestion
通常，在应用程序中会有许多 UI 组件，每个组件都包含不同的状态。
```

File: docusaurus/docs/developing-components-in-isolation.md [hunk 3]
```diff
@@ -1,59 +1,57 @@
...
-- In a loading state.
+- 一般都会有的 label 文本。
+- 禁止状态
+- 加载状态
 
-Usually, it’s hard to see these states without running a sample app or some examples.
+通常，如果不运行应用程序或一些示例，很难看到这些状态。
 
-Create React App doesn’t include any tools for this by default, but you can add [Storybook for React](https://storybook.js.org) ([source](https://github.com/storybooks/storybook)) or [React Styleguidist](https://react-styleguidist.js.org/) ([source](https://github.com/styleguidist/react-styleguidist)) to your project. **These are third-party tools that let you develop components and see all their states in isolation from your app**.
+Create React App 默认不包含任何查看组件状态的工具，但可以在项目里加入[Storybook for React](https://storybook.js.org) ([源码](https://github.com/storybooks/storybook)) 或 [React Styleguidist](https://react-styleguidist.js.org/) ([源码](https://github.com/styleguidist/react-styleguidist)) **这些第三方工具可以让你开发组件，并独立于应用程序查看它们的所有状态**.
```
Comment (QC-L) [inline docusaurus/docs/developing-components-in-isolation.md]:
```suggestion
Create React App 默认不包含任何查看组件状态的工具，但可以在项目里加入 [Storybook for React](https://storybook.js.org) ([源码](https://github.com/storybooks/storybook)) 或 [React Styleguidist](https://react-styleguidist.js.org/) ([源码](https://github.com/styleguidist/react-styleguidist)) **这些第三方工具可以协助你开发组件，并能够独立于应用程序查看它们所有的状态**.
```

Comment (QC-L) [reply docusaurus/docs/developing-components-in-isolation.md]: 注意空格

File: docusaurus/docs/developing-components-in-isolation.md [hunk 4]
```diff
@@ -1,59 +1,57 @@
...
 
-![Storybook for React Demo](https://i.imgur.com/7CIAWpB.gif)
+![Storybook React示例](https://i.imgur.com/7CIAWpB.gif)
```
Comment (QC-L) [inline docusaurus/docs/developing-components-in-isolation.md]:
```suggestion
![Storybook React 示例](https://i.imgur.com/7CIAWpB.gif)
```

File: docusaurus/docs/developing-components-in-isolation.md [hunk 5]
```diff
@@ -1,59 +1,57 @@
...
-You can also deploy your Storybook or style guide as a static app. This way, everyone in your team can view and review different states of UI components without starting a backend server or creating an account in your app.
+也可以将 Stroybook 或样式指南部署为静态应用程序。这样，团队中的每个人都可以查看和检查 UI 组件的不同状态，而无需启动后端服务器或在应用程序中创建帐户。
 
-## Getting Started with Storybook
+## Storybook 入门
 
-Storybook is a development environment for React UI components. It allows you to browse a component library, view the different states of each component, and interactively develop and test components.
-
-Run the following command inside your app’s directory:
+Storybook 是 React UI 组件的开发环境。它允许你浏览组件库，查看每个组件的不同状态，并交互式地开发和测试组件。
```
Comment (QC-L) [inline docusaurus/docs/developing-components-in-isolation.md]:
```suggestion
Storybook 是 React UI 组件的开发环境。它允许你浏览组件库，查看每个组件的不同状态，并进行交互式地开发以及组件测试。
```

File: docusaurus/docs/developing-components-in-isolation.md [hunk 6]
```diff
@@ -1,59 +1,57 @@
...
+- [学习 Storybook (教程)](https://learnstorybook.com)
+- [文档](https://storybook.js.org/basics/introduction/)
+- [GitHub 地址](https://github.com/storybooks/storybook)
+- [Storybook 快照测试 UI](https://github.com/storybooks/storybook/tree/master/addons/storyshots) + 插件/快照
 
-Styleguidist combines a style guide, where all your components are presented on a single page with their props documentation and usage examples, with an environment for developing components in isolation, similar to Storybook. In Styleguidist you write examples in Markdown, where each code snippet is rendered as a live editable playground.
+## Styleguidist 入门
 
-First, install Styleguidist:
+Styleguidist 类似于 Storybook，有一个独立开发组件的环境，之外，结合了一个样式指南，每个组件包含它的属性文档和使用示例都展示在单个页面里。同时，你也可以用Markdown编写示例，在Markdown中，每个代码片段都呈现为可编辑的区域。
```
Comment (QC-L) [inline docusaurus/docs/developing-components-in-isolation.md]:
```suggestion
Styleguidist 类似于 Storybook，有一个独立开发组件的环境，此外，它集成了样式指南，每个组件包含的属性文档和使用示例都会展示在单页面中。同时，你也可以用 Markdown 编写示例，在 Markdown 中，每个代码片段都呈现为可编辑的区域。
```

File: docusaurus/docs/developing-components-in-isolation.md (hunk 3 above)
Comment (liweina-fans) [reply docusaurus/docs/developing-components-in-isolation.md]: 嗯嗯 收到

**Pull request merged**